README.md

```diff
@@ -21,6 +22,26 @@ To use [Google Cloud Pub/Sub](https://cloud.google.com/pubsub/), you require a [
 
 For the list of command line flags, see [here](export/src/main/java/com/google/cloud/healthcare/imaging/dicomadapter/Flags.java)
 
+## Stackdriver Monitoring
+
+Both Import and Export adapter include Stackdriver Monitoring. Export adapter [events](export/src/main/java/com/google/cloud/healthcare/imaging/dicomadapter/monitoring/Event.java), Import adapter [events](import/src/main/java/com/google/cloud/healthcare/imaging/dicomadapter/monitoring/Event.java).
+Monitored resource is configured as k8s_container, with values set from combination of environment variables configured via DownwardAPI(pod name, pod namespace and container name) and GCP Metadata (project id, cluster name and location). Defaults to global resource, if k8s_container can't be configured.
```

Comment:
Please replace "Defaults to global resource" with
"Defaults to the global resource"

Reply:
Ok